tests/scripts/test_psa_constant_names.py

```diff
@@ -324,7 +324,18 @@ def normalize(expr):
     """
     return re.sub(NORMALIZE_STRIP_RE, '', expr)
 
-def do_test(options, inputs, type_word, names):
+def collect_values(options, inputs, type_word):
+    """Generate expressions using known macro names and calculate their values.
+
+    Return a list of pairs of (expr, value) where expr is an expression and
+    value is a string representation of its integer value.
+    """
+    names = inputs.get_names(type_word)
+    expressions = sorted(inputs.generate_expressions(names))
+    values = run_c(options, type_word, expressions)
+    return expressions, values
+
+def do_test(options, inputs, type_word):
```

Comment:
In commit "Move value collection into its own function", which says it makes no change to behavior, don't we need to update all callers of `do_test()`? We are changing the parameters `do_test()` makes in a commit that doesn't change the callers, so I don't see how this wouldn't change behavior.